docs/proposals/all-in-one-volume.md

```diff
@@ -0,0 +1,231 @@
+## Abstract
+
+A proposal for enabling users to create a volume populated from secrets,
+configmaps, and downward API.
+
+## Motivation
+
+This new feature will allow the user a method of combining data from different
```

Comment:
Users often need to build directories that contain multiple types of configuration and secret data.  For example, a configuration directory for some software package may contain both config files and credentials.  Currently, there is no way to achieve this in Kubernetes without scripting inside of a container.